### ISSUE TITLE
Support for different default language in l10n

### ISSUE THREAD

**Issue by jcolladosp** [microsoft/vscode#166939]
Hi,
I checked that bundle.l10n.json can only be set in English.
In our extension, the strings in the code are in Spanish, like this:

`const message = l10n.t('Hola');
`

This generates a bundle.l10n.json in Spanish, and we want to have a bundle.l10n.en.json translated in English.
But this doesn't work, since for English language it doesn't take the strings from bundle.l10n.en.json (it takes them from the l10n.t text)

In other words, if I have a file  `bundle.l10n.en.json` doesn't work if my vscode is in english. Instead, it takes the string from l10n.t().

Do you have any solution for this?

Thank you very much
